lib/zooniverse_data/projects/wise.rb

```diff
@@ -0,0 +1,21 @@
+module ZooniverseDaya
+  module Projects
+    class Wise
+      include Helpers
+
+      def customize_subject
+        new_locations = {}
+
+        entry.location['standard'].each do |path|
+          image_type = path.match(/.*_(?<image_type>[a-zA-Z0-9]+)\.png$/)
+          image_type = image_type[:image_type]
+          new_locations[image_type] = path
+        end
+
+        new_locations['standard'] = new_locations['wise4']
```

Comment:
Yup, that’s intentional. For some reason that’s how it is in the current data set.

> On 26 Apr 2016, at 17:01, Michael Parrish <notifications@github.com> wrote:
> 
> In lib/zooniverse_data/projects/wise.rb:
> 
> > @@ -0,0 +1,21 @@
> > +module ZooniverseDaya
> > +  module Projects
> > +    class Wise
> > +      include Helpers
> > +
> > +      def customize_subject
> > +        new_locations = {}
> > +
> > +        entry.location['standard'].each do |path|
> > +          image_type = path.match(/.*_(?<image_type>[a-zA-Z0-9]+)\.png$/)
> > +          image_type = image_type[:image_type]
> > +          new_locations[image_type] = path
> > +        end
> > +
> > +        new_locations['standard'] = new_locations['wise4']
> 
> It looks like this will duplicate the location (i.e. wise4 will duplicate standard). Is that okay or did you mean
> 
> new_locations['standard'] = new_locations.delete 'wise4'
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>